pkg/registry/core/service/storage/storage_test.go

```diff
@@ -614,3 +616,4562 @@ func TestServiceDefaulting(t *testing.T) {
 		})
 	}
 }
+
+func fmtIPFamilyPolicy(pol *api.IPFamilyPolicyType) string {
+	if pol == nil {
+		return "<nil>"
+	}
+	return string(*pol)
+}
+
+func fmtIPFamilies(fams []api.IPFamily) string {
+	if fams == nil {
+		return "[]"
+	}
+	return fmt.Sprintf("%v", fams)
+}
+
+func familyOf(ip string) api.IPFamily {
+	if netutils.IsIPv4String(ip) {
+		return api.IPv4Protocol
+	}
+	if netutils.IsIPv6String(ip) {
+		return api.IPv6Protocol
+	}
+	return api.IPFamily("unknown")
+}
+
+// Prove that create ignores IPFamily stuff when type is ExternalName.
+func TestCreateIgnoresIPFamilyForExternalName(t *testing.T) {
```

Comment:
note for others, this disappears later or at least I can't find it

Reply:
renamed to `TestCreateIgnoresIPsForExternalName`